app/src/main/java/org/p2p/wallet/swap/interactor/serum/SerumSwapInteractor.kt

```diff
@@ -717,6 +716,7 @@ class SerumSwapInteractor(
             signers.add(0, Account(tokenKeyProvider.secretKey))
         }
 
+// TODO: DELETE?
```

Comment:
need help, what should i do here? it warns me that isSimulation parameter is not used, but its used in the commented code